.github/workflows/php.yml

```diff
@@ -10,11 +10,16 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
-
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       fail-fast: false
       matrix:
         php-versions: ['7.2', '7.3', '7.4']
+        experimental: [false]
+        include:
+          - php-versions: '8.0'
```

Comment:
adds PHP 8.0 but allows it to fail if a new release comes out and we're not tracking it too closely.